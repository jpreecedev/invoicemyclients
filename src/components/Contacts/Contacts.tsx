import React from 'react'

const Contacts = () => {
  return (
    <form>
      <div className="form-group narrow d-flex">
        <label htmlFor="firstName" className="w-25 d-flex align-items-center">
          <small>First Name</small>
        </label>
        <input type="text" className="form-control" id="firstName" placeholder="First Name" />
      </div>
      <div className="form-group narrow d-flex">
        <label htmlFor="lastName" className="w-25 d-flex align-items-center">
          <small>Last Name</small>
        </label>
        <input type="text" className="form-control" id="lastName" placeholder="Last Name" />
      </div>
      <div className="form-group narrow d-flex">
        <label htmlFor="email" className="w-25 d-flex align-items-center">
          <small>Email</small>
        </label>
        <input type="text" className="form-control" id="email" placeholder="Email" />
      </div>
      <div className="form-group narrow d-flex">
        <label htmlFor="phoneNumber" className="w-25 d-flex align-items-center">
          <small>Phone Number</small>
        </label>
        <input type="text" className="form-control" id="phoneNumber" placeholder="Phone Number" />
      </div>
    </form>
  )
}

export { Contacts }
