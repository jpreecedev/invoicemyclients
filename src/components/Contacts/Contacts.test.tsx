import React from 'react'
import { render } from '@testing-library/react'
import { Contacts } from './Contacts'

describe('<Contacts /> tests', () => {
  it('should match snapshot', () => {
    expect(render(<Contacts />).container).toMatchSnapshot()
  })
})
