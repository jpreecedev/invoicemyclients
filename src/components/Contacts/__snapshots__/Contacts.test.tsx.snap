// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contacts /> tests should match snapshot 1`] = `
<div>
  <form>
    <div
      class="form-group narrow d-flex"
    >
      <label
        class="w-25 d-flex align-items-center"
        for="firstName"
      >
        <small>
          First Name
        </small>
      </label>
      <input
        class="form-control"
        id="firstName"
        placeholder="First Name"
        type="text"
      />
    </div>
    <div
      class="form-group narrow d-flex"
    >
      <label
        class="w-25 d-flex align-items-center"
        for="lastName"
      >
        <small>
          Last Name
        </small>
      </label>
      <input
        class="form-control"
        id="lastName"
        placeholder="Last Name"
        type="text"
      />
    </div>
    <div
      class="form-group narrow d-flex"
    >
      <label
        class="w-25 d-flex align-items-center"
        for="email"
      >
        <small>
          Email
        </small>
      </label>
      <input
        class="form-control"
        id="email"
        placeholder="Email"
        type="text"
      />
    </div>
    <div
      class="form-group narrow d-flex"
    >
      <label
        class="w-25 d-flex align-items-center"
        for="phoneNumber"
      >
        <small>
          Phone Number
        </small>
      </label>
      <input
        class="form-control"
        id="phoneNumber"
        placeholder="Phone Number"
        type="text"
      />
    </div>
  </form>
</div>
`;
